### ISSUE TITLE
Missing ability to access built older versions of the WebAssembly specification in archivable form

### ISSUE THREAD

**Issue by 20kdc** [WebAssembly/spec#1736]
There does not appear to be a simple way to access built older versions of the WebAssembly specification, particularly 1.0, in archivable form (the various builds, i.e. PDF etc.)

I was at least able to find the W3C version of the core specification ( https://www.w3.org/TR/wasm-core-1/# ), but that appears to be all, and it doesn't appear to have any standardized archival download readily available.

## Why care about 1.0?

There are two main use-cases I can think of for the WebAssembly 1.0 specification:

* With the addition of extensions/etc. onto the "MVP" version of WebAssembly in the core specification (for example: `v128`), the support for these in browsers that are actually available on end-user devices will vary. There will be, for quite some time, the need to maintain WebAssembly 1.0-compliant versions of ecosystem software.
* Non-browser targets (i.e. WASI) may use custom WebAssembly runtimes that may not have use for functionality such as vectorization. Alternatively, non-browser usecases may require targetting outdated runtimes. These usecases benefit from being able to point at a specific specification version as the standard.
    * A practical example would be https://github.com/dtolnay/watt, which contains a WebAssembly interpreter. This interpreter seems to be intended solely for the purpose of running compiled Rust code handling abstract token streams. This is reliant on upstream tooling supporting an unextended WebAssembly 1.0 environment -- it would break if tooling support for WebAssembly 1.0 were to begin failing (i.e. by including WebAssembly 2.0 features in code intended to target WebAssembly 1.0).

It is thus important to keep available the specification versions that actually apply to these targets.

## Workaround

A present workaround is to use the mirrors generated by WebAssembly project proposals that were forked from late-stage 1.0, i.e. https://webassembly.github.io/multi-value/core/_download/WebAssembly.pdf -- _however, there are still some differences, so this is probably not a definitive 1.0._

**Comment by Ms2ger** [comment]
Going forward, all new versions will be available at https://www.w3.org/standards/history/wasm-core-2/

**Comment by rossberg** [comment]
I agree they should be made available here (especially since the w3c site will probably only have the inferior single-page HTML version). We have talked about this in the past, but never took action.

FWIW, you should be able to at least build the old releases of the spec by checking out this repo at specific release tags, e.g., `wg-1.0` for Wasm 1.0. However, there may have been some bitrot, as both Sphinx and Bikeshed have a tendency to break our setup when they release major new versions (though you can probably ignore the latter).
